### DISCUSSION TITLE
Suggestion Window Position

### DISCUSSION THREAD

**Discussion by ASHspace** [OpenBangla/OpenBangla-Keyboard#233] [Q&A]
My system is **Manjaro KDE**. When I try to type in **Notepadqq**, **Joplin**, etc. app, the suggestion box appears on the top of the typing text. So typing is not clearly visible. I tried to move it a little bit down but it appears again on top of the typing text. How can I change the position of the suggestion window in the KDE environment?

**Comment by mominul** [comment]
Are you using version 2.0.0 of OpenBangla Keyboard and installed `manjaro-asian-input-support` aur package? If not please follow these instructions and log out from your session:
```
git clone https://aur.archlinux.org/manjaro-asian-input-support.git
cd manjaro-asian-input-support
makepkg
pacman -U ibus-input-support-*.pkg.tar.zst
```

Thanks!

**Reply by ASHspace** [reply]
Yes, I am using OBK 2.0 and `manjaro-asian-input-support` installed system. It is working fine for LibreOffice but some applications are showing overlapped suggestion window.

Thanks.

**Reply by ahmubashshir** [reply]
এই বাগ আমারো হয়েছে... শুধু কিউটি অ্যাপেই।

**Reply by ahmubashshir** [reply]
`env | grep 'ibus'` রান করে দেখেন `QT_IM_MODULE` আইবাস কিনা।

**Reply by mominul** [reply]
It's not an OBK's bug from the start, it's a problem between ibus and the application. Can you try running `notepadqq` from the terminal by this command and check if the problem is fixed?:
```
QT_IM_MODULE=ibus notepadqq
```

**Reply by ASHspace** [reply]
> It's not an OBK's bug from the start, it's a problem between ibus and the application. Can you try running `notepadqq` from the terminal by this command and check if the problem is fixed?:
> 
> ```
> QT_IM_MODULE=ibus notepadqq
> ```

Still same problem.
If it is app and ibus problem, do you know any workaround coz this is affecting several applications?

**Reply by ASHspace** [reply]
> `env | grep 'ibus'` রান করে দেখেন `QT_IM_MODULE` আইবাস কিনা।

`ibus` already set.

**Reply by mominul** [reply]
@getashraf Sorry, I can't think of a workaround for this problem. But you can try reporting issues to [notepadqq](https://github.com/notepadqq/notepadqq) and other application's issue pages to get help.

But there is an alternative approach you can try, we are creating a new OBK frontend that uses the fcitx5 input method framework as the backend. You can try installing the [`fcitx5-openbangla-git`](https://aur.archlinux.org/packages/fcitx5-openbangla-git) and [`fcitx5-input-support`](https://aur.archlinux.org/packages/fcitx5-input-support/) AUR packages while uninstalling ibus and OBK 2.0.0 version packages. Please note that this is in the development stage so expect bugs and breaking changes!
@ahmubashshir @chandradeepdey Can you provide detailed installation so that @getashraf could follow?

**Reply by ASHspace** [reply]
Thanks for the suggestion.

**Comment by ahmubashshir** [comment]
If you use aur helpers, install fcitx5-openbangla-git. that's enough to 
install fcitx5 frontend.
	Mubashshir
